packages/oauth/oauth-client-node/README.md

```diff
@@ -415,9 +419,15 @@ app.get('/atproto-oauth-callback', async (req, res) => {
   const params = new URLSearchParams(req.url.split('?')[1])
   try {
     try {
-      const { agent, state } = await client.callback(params)
+      const { session, state } = await client.callback(params)
+
+      // Process successful authentication here. For example:
+
+      const agent = new Agent(session)
+
+      const profile = await agent.getProfile({ actor: agent.did })
 
-      // Process successful authentication here
+      console.log('Bsky profile:', profile.data)
```

Comment:
Was this change intentional or part of debugging?  All good either way, just checking!

Reply:
Intentional (this is just example code)